internal/playwright/config.go

```diff
@@ -141,3 +147,40 @@ func SplitSuites(p Project) (Project, Project) {
 
 	return dockerProject, sauceProject
 }
+
+func checkSupportedBrowsers(p *Project) error {
+	supportedBrwsList := []string{"chromium", "firefox", "webkit"}
```

Comment:
Might be better to declare this var at the package level instead.

Reply:
I moved this var.